sys/riscv/riscv/elf_machdep.c

```diff
@@ -255,6 +257,9 @@ static const struct type2str_ent t2s[] = {
 	{ R_RISCV_HI20,		"R_RISCV_HI20"		},
 	{ R_RISCV_LO12_I,	"R_RISCV_LO12_I"	},
 	{ R_RISCV_LO12_S,	"R_RISCV_LO12_S"	},
+#ifdef __CHERI_PURE_CAPABILITY__
```

Comment:
Probably does no harm to enable this unconditionally (or at least with `__has_feature(capabilities)`) if it's for printing out debug info?

Reply:
The only use is under the same #ifdef later in the code.